light-client/src/light_client.rs

```diff
@@ -168,8 +164,8 @@ impl LightClient {
             // Get the highest trusted state
             let trusted_state = state
                 .light_store
-                .highest(VerifiedStatus::Verified)
-                .ok_or_else(|| ErrorKind::NoInitialTrustedState)?;
+                .highest(Status::Verified)
```

Comment:
@romac  please confirm that the following explanation makes sense and I can open an issue. Give that the light store does not have a notion that `Trusted` blocks are also `Verified` we will probably always run into a situation where we immediately fail to do `verify_to_target` as the only block we have would be a block which is `Trusted`. Also, it will happen that after one run of `Supervisor` verification the highest block we'll get from here is not the one which was previously the target as that block is in the `Trusted` state. It might also make sense to update single peer tests to start of with `Trusted` light blocks as that would immediately show that we can not do correct verification if the only block we have is trusted.

Reply:
Yeah you are right, this should actually fetch the latest verified or trusted block from the store. Will fix!

Reply:
Fixed in https://github.com/informalsystems/tendermint-rs/pull/407